src/components/shared/Accordion.css

```diff
@@ -54,6 +54,7 @@
 .accordion ._content {
   border-bottom: 1px solid var(--theme-splitter-color);
   font-size: 12px;
+   padding: 3.5px;
```

Comment:
any need for `.5`? is `4px` not good?

Reply:
Nice @EmdaviesSeneca -- I can't update this PR because you've done this work on master branch, but once you update this to `4px` we should be good :)